src/components/molecules/HistoricalChart/HistoricalChart.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryFn } from '@storybook/react';
+import { tradingData } from 'src/stories/mocks/historicalchart';
+import { mockTransactionCandleStick } from 'src/stories/mocks/queries';
+import { HistoricalChart } from '.';
+
+export default {
+    title: 'Molecules/HistoricalChart',
+    component: HistoricalChart,
+    args: {
+        className: 'font-bold w-[600px]',
```

Comment:
what is the significance of this className here?

Reply:
I removed `font-bold`. 600px width is for showing the historical chart in a decent-sized container like its done in HistoricalWidget